src/main/java/com/github/rodionovsasha/commentservice/services/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,70 @@
+package com.github.rodionovsasha.commentservice.services.impl;
+
+import com.github.rodionovsasha.commentservice.entities.User;
+import com.github.rodionovsasha.commentservice.exceptions.InactiveUserException;
+import com.github.rodionovsasha.commentservice.exceptions.UserNotFoundException;
+import com.github.rodionovsasha.commentservice.repositories.UserRepository;
+import com.github.rodionovsasha.commentservice.services.UserService;
+import lombok.AllArgsConstructor;
+import lombok.val;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.function.Consumer;
+
+@AllArgsConstructor
+@Service @Transactional
+public class UserServiceImpl implements UserService {
+    private final UserRepository repository;
+
+    @Override
+    public User create(String name, int age) {
+        return repository.save(new User(name, age));
+    }
+
+    @Override
+    @Transactional(readOnly = true)
+    public User getById(long id) {
+        val user = repository.findOne(id);
+        if (user == null) {
+            UserNotFoundException.withId(id);
```

Comment:
This place is still confusing. Imo, `withId` should build and return the exception but it should not throw it, it's just a builder.

Reply:
fixed